doc/modules/model_evaluation.rst

```diff
@@ -1034,6 +1034,16 @@ note that in the binary case,
 so expanding the inner sum over :math:`y_{i,k} \in \{0,1\}`
 gives the binary log loss.
 
+As a special case when there are more than one label per sample
```

Comment:
I think it would be simpler to say that the multilabel case is measured as the mean of binary log-loss measurements for each class, without another formula

Reply:
sure.